script/bootstrap.sh

```
@@ -3,6 +3,26 @@
 if [ -z "$RUN_ENV" ]; then
     export RUN_ENV="prod"
 fi
+
+CURDIR=$(cd $(dirname $0); pwd)
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)